ComponentTests/__init__.py

```diff
@@ -0,0 +1,5 @@
+from Condenser import CondenserClass
```

Comment:
@davideziviani Do you think we need this? 
I think we can also remove it from the main source code.